app/models/user.rb

```diff
@@ -1,8 +1,16 @@
 class User < ActiveRecord::Base
 
+  attr_accessor :password_confirmation
```

Comment:
`:password_confirmation` is provided by `has_secure_password` if you turn on validations.